app.py

```diff
@@ -33,7 +33,7 @@ def create_app():
         scope="user-library-read user-top-read playlist-modify-private",
         open_browser=False,
     )
-    spotify = spotipy.Spotify(auth_manager=auth_manager)
+    spotify = spotipy.Spotify(auth_manager=auth_manager, requests_timeout=45, retries=0)
```

Comment:
may or may not want to include this.  Was trying to troubleshoot but I think I was just getting throttled by spotify